charts/licensify/values.yaml

```diff
@@ -18,6 +18,22 @@ clamav:
     type: ClusterIP
     port: 3310
 
+nginx:
+  port: 8080
+  proxyConnectTimeout: 30s
+  proxyReadTimeout: 15s
+  clientMaxBodySize: 200M
+  extraHttpConf: |
+    limit_req_zone $binary_remote_addr zone=licensing:1m rate=1r/s;
+    limit_req_status 429;
```

Comment:
We can drop this too (as long as we stick the AWS WAF config on the load balancer).